app/src/main/java/com/codersgroup/furryfriendfinder/ui/theme/Dimmen.kt

```diff
@@ -0,0 +1,9 @@
+package com.codersgroup.furryfriendfinder.ui.theme
+
+import androidx.compose.ui.unit.dp
+
+val tinyPadding = 2.dp
+val smallPadding = 4.dp
+val mediumPadding = 8.dp
+val largePadding = 12.dp
+val hugePadding = 16.dp
```

Comment:
Mi sugerencia es utilizar paddings más genéricos, 8, 16, 24, 32 y como mínimo 4 en casos muy particulares.

Reply:
los que creáis adecuados, yo he puesto esos un poco como guía, pero dejamos los que hagan falta